src/server/game/Movement/MovementGenerators/PointMovementGenerator.cpp

```diff
@@ -142,8 +142,6 @@ void EffectMovementGenerator::Finalize(Unit* unit)
     {
         if (Unit* victim = unit->GetVictim())
             unit->GetMotionMaster()->MoveChase(victim);
-        else
-            unit->GetMotionMaster()->Initialize();
```

Comment:
Thanks, that would explain why pathing breaks for creatures reaching jump location.